modules/tinymce/src/themes/silver/test/ts/browser/editor/sizing/ResizeTest.ts

```diff
@@ -34,6 +35,16 @@ describe('browser.tinymce.themes.silver.editor.sizing.ResizeTTest', () => {
     });
   });
 
+  // Make sure the height is reset
+  beforeEach(() => {
+    const editor = hook.editor();
+    const container = SugarElement.fromDom(editor.getContainer());
+    Css.setAll(container, {
+      width: '400px',
```

Comment:
Should this use `editor.settings.width + 'px'` instead to remove duplication? or alternatively should we have a constant for the initial height/width?